src/components/GlobalPreferences/Notifications/NotificationsVerify.js

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect, useState } from 'react'
```

Comment:
If I type: `http://localhost:3000/#/dev/apps?preferences=/notifications/verify/asdas` into the URL box this renders the error box, not saying users will do that, but an error in the email or someplace in the flow might redirect with an incorrect token and this will happen.

Reply:
I guess that depends on the likelihood of links not working in emails because that's the only place where the user will see the link. 

Do you think providing the user with the option to copy the token in is a good idea?

Reply:
Hmmm, maybe, I recall when receiving emails lately they always include the "here's the link, and if that does not work copy and paste this into a browser" <- that last part could be where the user did not copy the full thing. Not saying major changes here, just validate the full token and if there's an error provide an error screen, somewhere in the lines of "Could not verify sign in"

Reply:
I understand.

So there are two potential error states:
- Problem validating/communicating with the API
- The token is invalid (incomplete/not properly copied) which we could loosely validate client side

For the second we could add a more nuanced error